helix-view/src/input.rs

```diff
@@ -387,6 +387,21 @@ impl std::str::FromStr for KeyEvent {
                     .then_some(KeyCode::F(function))
                     .ok_or_else(|| anyhow!("Invalid function key '{}'", function))?
             }
+            _ if s.ends_with('-')
+                && tokens.last().map(|s| str::is_empty(s)).unwrap_or_default() =>
```

Comment:
```suggestion
                && tokens.last().map_or(false, |s| s.is_empty()) =>
```

Reply:
this is now gone from applying the previous suggestion